pkg/cluster/machine/machine_test.go

```diff
@@ -76,7 +75,8 @@ const machinesValid = `
   metadata:
     name: master-0
   spec:
-    address: "172.17.8.101"
+    private:
+      address: "172.17.8.101"
```

Comment:
For the record this became an error after 8408805db4333cf045f85758e461bd49750e3fd3 and I missed the cleanup.